themes/StarterTheme/templates/catalog/product-miniature.tpl

```diff
@@ -5,13 +5,14 @@
       <img
         src = "{$product.cover.medium.url}"
         alt = "{$product.cover.legend}"
+        title = "{$product.cover.legend}"
```

Comment:
Title tag has a limitation in characters. It should not duplicate the alt tag content. Perhaps modifying the Back Office and a new record field.